### ISSUE TITLE
Allow setting path to test suites

### ISSUE THREAD

**Issue by MelvIsntNormal** [elm-tooling/elm-language-server#497]
## Expected Behavior

The extension should allow you to specify which files are test suites.

## Current Behavior

The extension assumes that anything in the `test` directory is a test suite, but does not recognise any other files as such.

## Possible Solution

I understand that there's some talk about adding an option to `elm-tooling.json` to set this. In the absence of that feature, I'd like to be able to set the option in the Preferences tab in VS Code, similar to how we can set the path for `elm-test` and `elm-analyze`.

## Steps to Reproduce (for bugs)

Not applicable.

## Context

I currently work on a project where we prefer to put our test suites next to the module in question: for `Action.elm` we may have `ActionTest.elm` in the same directory. To run tests, we use `elm-test "src/**/*Test.elm"` and it works brilliantly. It would be cool if we could configure this extension to recognise test files in the same way.

## Your Environment

_From `Help > About`_
* Version: 1.52.0-insider
* Commit: ee46c1f89ebb213e0827886f1688c0fbb55be49f
* Date: 2020-11-13T06:46:37.453Z
* Electron: 9.3.3
* Chrome: 83.0.4103.122
* Node.js: 12.14.1
* V8: 8.3.110.13-electron.0
* OS: Linux x64 5.9.8-arch1-1

**Comment by Razzeee** [comment]
Current consensus is, that it would be better to automatically detect these based on types.
See: https://github.com/elm-tooling/elm-tooling-cli/issues/17#issuecomment-834098781

We've also aligned some code, so that there is one source of truth now. We now "only" need to write that detection logic and everything else should fall into place.

**Comment by MelvIsntNormal** [comment]
Hi guys, it's been a couple of years since I made this issue and I'm wondering if there's any progress on this front? Is there anything I could contribute? Clearly the issue isn't really blocking me, but it would be nice to not have a sea of errors in VS Code :sweat_smile:

**Comment by razzeee** [comment]
I don't think anyone worked on this. If you want to look into this, feel free to have a go.

**Comment by MelvIsntNormal** [comment]
Okay, so just to make sure, the consensus is still to do automatic test detection rather than to allow a path to be configured for tests?

**Comment by razzeee** [comment]
Yeah, in fact I'm unsure, if the test view shouldn't already detect your tests fine. And it's just the other parts of the codebase.

**Comment by MelvIsntNormal** [comment]
My test files are not being recognised; they're not in a `tests` folder, but they live next to the module they're testing. `<Module>.elm` would have a `<Module>Test.elm` file next to it
